use-shopping-cart/core/slice.js

```diff
@@ -90,6 +90,7 @@ const slice = createSlice({
         state.cartCount = 0
         state.totalPrice = 0
         state.cartDetails = {}
+        state.formattedTotalPrice = ''
```

Comment:
That doesn't look like what you might want it to be. If you do this, they will no longer have a `formattedTotalPrice`. When we clear the cart we are resetting the values. Upon changing the `totalPrice`, the `currency`, or the `language`, we will want to update the `formattedTotalPrice` to reflect those values.

```suggestion
        updateFormattedTotalPrice(state)
```